config/helm/chart/default/templates/Common/webhook/deployment-webhook.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
   labels:
     {{- include "dynatrace-operator.webhookLabels" . | nindent 4 }}
 spec:
-  replicas: 1
+  replicas: {{ (.Values.webhook).enableHighAvailability | ternary 2 1 }}
```

Comment:
I think the ternary operator is less readable, but that's just a matter of preference. 
If it stays that way I am also fine with it